pom.xml

```diff
@@ -44,8 +44,8 @@
         <maven.compiler.release>11</maven.compiler.release>
 
         <!-- DEPENDENCY PROPERTIES -->
-        <spec.version>2.1.0</spec.version>
-        <tck.version>2.1.0</tck.version>
+        <spec.version>3.0.0.M1</spec.version>
+        <tck.version>3.0.0.M1</tck.version>
 
         <jersey.version>3.1.0-M8</jersey.version>
```

Comment:
Update jersey version to 3.1.1 ?

Reply:
Yes good catch. Just focused on the spec yesterday. I'll change it

Reply:
Done